### PR TITLE
[Reviewer: Matt] Updated MIB file for Chronos total timers count

### PR DESCRIPTION
**Pull request by TygerBurning**
Updated to MIB file for Chronos total timers count

### CODE REVIEW

File: PROJECT-CLEARWATER-MIB [hunk 1]
```diff
@@ -4500,7 +4500,8 @@ chronosRemainingNodesQuery OBJECT-TYPE
     SYNTAX      Unsigned32
     MAX-ACCESS  read-only
     STATUS      current
-    DESCRIPTION "Current count of the remaining Chronos nodes to query during a scaling operation."
+    DESCRIPTION "Current count of the remaining Chronos nodes to query during a
+scaling operation."
```
Comment (mirw) [inline PROJECT-CLEARWATER-MIB]: I think this should be indented, if it needs to go on a new line at all.

File: PROJECT-CLEARWATER-MIB [hunk 2]
```diff
@@ -4579,11 +4580,94 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
     DESCRIPTION "The total number of invalid timers processed over the period."
     ::= { chronosInvalidTimersProcessedEntry 2 }
 
+chronosTotalTimersTable OBJECT-TYPE
+    SYNTAX SEQUENCE OF ChronosTotalTimersEntry
+    MAX-ACCESS         not-accessible
+    STATUS             current
+    DESCRIPTION        "The total number of timers this Chronos node
+                       is managing - including primary and backup
+                       timers"
```
Comment (mirw) [inline PROJECT-CLEARWATER-MIB]: Strictly speaking, "primary and backup timers" aren't well-defined - I think what you mean are both timers for which this node is primary, and for which this node is backup.

File: PROJECT-CLEARWATER-MIB [hunk 3]
```diff
@@ -4579,11 +4580,94 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
...
+    DESCRIPTION "The lowest number of total timers stored
+                 over the period."
+    ::= { chronosTotalTimersEntry 5 }
+
+chronosCurrentTimers OBJECT-TYPE
+    SYNTAX      Unsigned32
+    MAX-ACCESS  read-only
+    STATUS      current
+    DESCRIPTION "The current number of timers (both primary
+                 and backup) that this Chronos is managing"
```
Comment (mirw) [inline PROJECT-CLEARWATER-MIB]: Same comment as above.

**Comment by mirw** [comment]
A few comments inline.  Otherwise looks fine.

**Comment by TygerBurning** [comment]
Fair comments, markups done

**Comment by mirw** [comment]
Fine.

**Pull request merged**